### PR TITLE
Removed a if statement from CollectCargo and DiscardCargo

### LINKED ISSUE #24: Solenoid state is checked in `CollectCargo` and `DiscardCargo` when it's not needed
Remove the if statement from the following
https://github.com/FRCTeam3255/Robot2022-V/blob/7419c6c4b3e5a443161a916f993e15fe87bd7a3f/src/main/java/frc/robot/commands/Cargo/CollectCargo.java#L71-L73

https://github.com/FRCTeam3255/Robot2022-V/blob/7419c6c4b3e5a443161a916f993e15fe87bd7a3f/src/main/java/frc/robot/commands/Cargo/DiscardCargo.java#L60-L62

### PR DESCRIPTION
**Pull request by Adam-Walls077**
closes #24

### CODE REVIEW

**Review by ACat701 — CHANGES_REQUESTED** [review]
Both commands still need to retract the intake. The issue is that the if statement is redundant, because if its already retracted, retracting it again won't do anything

**Review by TaylerUva — CHANGES_REQUESTED** [review]
To specify we need the internals of the if statement just not the it itself

File: src/main/java/frc/robot/commands/Cargo/CollectCargo.java [hunk 1]
```diff
@@ -68,9 +68,9 @@ public void execute() {
         outputEntrance = RobotPreferences.zeroDoublePref;
         outputBottom = RobotPreferences.zeroDoublePref;
 
-        if (subIntake.isDeployed()) {
-          subIntake.setRetracted();
-        }
+        subIntake.isDeployed();
```
Comment (TaylerUva) [inline src/main/java/frc/robot/commands/Cargo/CollectCargo.java:71]: This is a boolean so it doesn't do anything on its own

File: src/main/java/frc/robot/commands/Cargo/DiscardCargo.java [hunk 2]
```diff
@@ -57,9 +57,8 @@ public void execute() {
 
       RobotContainer.cargoState = CargoState.DISCARDING;
 
-      if (subIntake.isDeployed()) {
-        subIntake.setRetracted();
-      }
+      subIntake.isDeployed();
```
Comment (TaylerUva) [inline src/main/java/frc/robot/commands/Cargo/DiscardCargo.java:60]: This is a boolean so it doesn't do anything on its own

**Review by TaylerUva — CHANGES_REQUESTED** [review]
Remove the unused boolean calls

**Pull request closed without merging**